fiftyone/core/stages.py

```diff
@@ -3225,6 +3239,12 @@ def _params(cls):
                 "placeholder": "",
                 "default": "None",
             },
+            {
```

Comment:
Just a curious question, why do we return this json info list for the arguments & params? Who are the endpoint consumers of these? :)

Reply:
`_params()` is used by the App to render the view stages in the view bar